src/theme.ts

```diff
@@ -51,7 +51,7 @@ export const THEME = {
   successColor: PALETTE.green,
   warningColor: PALETTE.gold,
   errorColor: PALETTE.red,
-  infoColor: PALETTE.gray4,
+  infoColor: PALETTE.gray7,
```

Comment:
@mic-web is it safe to change this? With this change, we change in all colors in which infocolor is referenced in limes.

Reply:
In limes it doesn't make a change, as the `infoColor` is overwritten with the limes-frontend `theme`, passed to the `Provider`. It only changes this component library's default color.